src/pages/DatasetCatalog.js

```diff
@@ -101,6 +102,40 @@ export default function DatasetCatalog(props) {
     init();
   }, [getDatasets]);
 
+  useEffect( () => {
+    const doEnrichment = async() => {
+      spinnerService.showAll();
+      const theCurrentPage = filterToOnlySelected ? currentPageOnlySelected : currentPageAllDatasets;
+      const searchTable = (query) => (row) => {
+        if (query) {
+          let text = JSON.stringify(row);
+          return text.toLowerCase().includes(query.toLowerCase());
+        }
+        return true;
+      };
+      const visibleDatasets = filterToOnlySelected ? selectedDatasets : datasetList;
+      const results = visibleDatasets.filter(searchTable(searchDulText)).slice((theCurrentPage - 1) * pageSize, theCurrentPage * pageSize);
+      await Promise.all(results.map(async(dataset) => {
+        if (isNil(dataset.codeList)) {
+          if (!dataset.dataUse || isEmpty(dataset.dataUse)) {
+            dataset.codeList = 'None';
```

Comment:
I remember when you were demoing this you mentioned you were providing "None" when there are no dataUse restrictions. I was flipping through a dataset catalog and noticed a dataset that had an empty string displayed. Digging deeper I found that it had two restrictions listed as false, so there was still a "dataUse" property for that dataset. Should datasets such as these be displaying "None"?
**dataSetId: 441**
`    "dataUse": {
      "generalUse": false,
      "other": "false"
    },`
![Screen Shot 2022-11-23 at 11 10 06 AM](https://user-images.githubusercontent.com/59845076/203594499-ba2b4003-5481-43ca-a827-857afda96945.png)

Reply:
agreed. below this, i think it would just be:

```
            dataset.codeList = isEmpty(codes) ? 'None' : codes.join(', ');
```

Reply:
This one is interesting.  Excellent find @quazi-broad 
It seems this is a case where our code was changed, but existing values in dataUse were not.

For DUOS-000078, the dataUse object is: `dataUse:{generalUse: false, other: 'false'}`
When I generate a new entry similar to it (see DUOS-000612 on dev), the dataUse object is: `dataUse:{otherRestrictions: true, other: 'What should the data use abbreviation be for this case?'}`

otherRestrictions is key here.  It causes a proper abbreviation to be generated where our existing code doesn't for DUOS-000078.

I'm inclined to leave this code as is because the absence is highlighting a bug.  Is that okay with you @connorlbark ?  I'm going to see if I can fix inside of dataUseTranslation.

Reply:
BEFORE:

![Screen Shot 2022-11-23 at 5 18 02 PM](https://user-images.githubusercontent.com/111771148/203655518-731ce00f-044a-4a4e-b1ef-6a971da42dfc.png)

AFTER:
![Screen Shot 2022-11-23 at 5 18 40 PM](https://user-images.githubusercontent.com/111771148/203655611-24500032-897b-45f2-a7ba-2298c2bd80c6.png)

Reply:
> I'm inclined to leave this code as is because the absence is highlighting a bug.

I agree with this. In theory, there should never be a completely empty data use. That would be an error case in the data and possibly a validation error when processing a new DAR.